File: examples/01_helloworld/asset/script/main.lua
local font = sn.Font.new()
font:load(32)
function Update()
    if sn.Keyboard.isPressed(sn.Keyboard.ESCAPE) then
        sn.Script.load("main", ".")
    end
end

function Draw()
    sn.Graphics.drawText("Hello World!", font, sn.Vec2.new(0, 0), sn.Color.new(1.0), 32)
end

File: examples/01_helloworld/asset/script/main.py
import sinen as sn


def update():
    if sn.Keyboard.is_pressed(sn.Keyboard.ESCAPE):
        sn.Script.load("main", ".")


def draw():
    sn.Graphics.draw_text("Hello World!", sn.Vec2(0), sn.Color(1), 32)
